docs/adr/0035-resizable-local-queue-initial-capacity.md

```
@@ -0,0 +1,116 @@
+# ADR-035: Resizable local-queue initial capacity
+
+Status: Accepted
+
+- Date: 2026-08-28
+- Change class: [major] [arch]
+- Refs: `MOI-LOCAL-QUEUE-CAPACITY-036`, ADR-034
+
+### Context
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)